### DISCUSSION TITLE
FullScreen and/or PinchZoom

### DISCUSSION THREAD

**Discussion by maste9** [pixelfed/pixelfed-rn#86] [Ideas]
I find the pictures - especially the landscape oriented - rather small. It would be nice to have either an automatic fullscreen view that uses the best phone orientiation for displaying the picture (landscape -> horizontal; portrait -> vertical).

Another nice to have would be to zoom in the picture via pinching maybe combined with the mentioned full screen view.

**Comment by turtlerock** [comment]
Yes, we MUST have pinch and zoom! Our fingers are trained for it.

Awesome app! Keep up the great work!
